.projenrc.ts

```diff
@@ -50,6 +65,18 @@ const project = new awscdk.AwsCdkConstructLibrary({
     `@aws-cdk/integ-runner@${cdkVersion}-alpha.0`,
     `@aws-cdk/integ-tests-alpha@${cdkVersion}-alpha.0`,
   ],
+  eslintOptions: {
+    dirs: ['src', 'test'],
+    prettier: true,
+  },
+  prettier: true,
+  prettierOptions: {
+    settings: {
+      singleQuote: true,
+      printWidth: 100,
```

Comment:
The previous ESLint rule used 150. I think that's maybe a bit excessive, but 120 might be a good compromise.

```suggestion
      printWidth: 120,
```